parkings/api/common_permit.py

```diff
@@ -15,6 +15,10 @@ class Meta:
         fields = ['id', 'created_at', 'modified_at', 'active']
         read_only_fields = fields
 
+    def validate(self, data):
+        data['owner'] = self.context['request'].user
```

Comment:
I think there should be some better place to do this, since the owner field should only be touched on creation, not on all validation. How about doing this in the `create` method?